packages/graphql-transformer-common/src/util.ts

```diff
@@ -1,9 +1,13 @@
 import md5 from 'md5';
+import pluralize from 'pluralize';
 
 export function plurality(val: string): string {
   if (!val.trim()) {
     return '';
   }
+  if (pluralize.isPlural(val.trim())) {
```

Comment:
It looks like this module can actually do the conversion to plural for you. Is there any reason not to use that functionality instead?

Side note: it might be worth putting `val.trim()` in a variable since it's called a few times now.

Reply:
Sure. I am pushing the changes soon.